generated/nifake/nifake_service.cpp

```diff
@@ -274,7 +274,7 @@ namespace nifake_grpc {
     try {
       auto vi_grpc_session = request->vi();
       ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
-      std::string a_string(256, '\0');
+      std::string a_string;
```

Comment:
Can you explain what this change is about? Same question for the change on line 878 below as well.

Seems like there is similar change in niscope_service.cpp and niswitch_service.cpp files as well.

Reply:
reverted those changes